package.json

```diff
@@ -13,7 +13,7 @@
     "lint": "tsc && next lint",
     "lint:fix": "yarn lint --fix",
     "prettier": "prettier -w {src,cypress,mocks,scripts}/**/*.{ts,tsx,css,js}",
-    "test": "jest",
+    "test": "DEBUG_PRINT_LIMIT=15000 jest",
```

Comment:
This ENV variable increases the maximum lines printed by `react-testing-library`, when the test fails. Because we use material Ui which produces a lot of nested divs, the default limit of 7000 isn't really handy